api/src/main/java/com/messagebird/objects/conversations/ConversationMessage.java

```diff
@@ -18,6 +19,7 @@
     private ConversationContent content;
     private Date createdDatetime;
     private Date updatedDatetime;
+    private Map<String, Object> source;
```

Comment:
Could you add `source` field to `ConversationMessageRequest` object as well please? If I dont see wrong, its missing

Reply:
Could you add `tag` property here please?

Ref: https://developers.messagebird.com/api/conversations/#the-message-object

Reply:
Done.